stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -110,6 +110,7 @@ public type Certificate record {|
 |};
 
 # Returns the MD5 hash of the given data.
+# ```ballerina byte[] hash = crypto:hashMd5(data); ```
```

Comment:
Shall we use the following pattern as discussed?
```suggestion
# ```ballerina
# byte[] hash = crypto:hashMd5(data);
```
```

Reply:
Changed the pattern.